cirq-aqt/cirq_aqt/aqt_device.py

```diff
@@ -11,18 +11,21 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+# pylint: disable=line-too-long
```

Comment:
We could consider configuring pylint to allow long lines that only contain urls. We have the following rule in our internal `.pylintrc`:
```
# Ignore long lines containing other pylint directives, type annotations, or urls.
ignore-long-lines=^(.*#\s*pylint: disable.*|.*#\s*type: .*|\s*(# )?<?https?://\S+>?)$
```
This could be tweaked here of course. And it may not be needed if we can use shorter markdown form for urls, as commented below.

Reply:
oh we seem to have this too, now investigating why it is failing

Reply:
oh I see this regex only works for case where url inside of brackets.

Reply:
updated regex